src/main.cpp

```diff
@@ -1,29 +1,31 @@
 #include "defines.h"
 #include "schnitzel_lib.h"
 #include "game.cpp"
+#include <stdio.h>
 
 // #############################################################################
 //                           Platform Globals
 // #############################################################################
-static bool running = true;
```

Comment:
Removing the static keyword from the running variable declaration allows 'running' to be shared and accessed from other C++ files, ensuring that changes made to the variable's state in one file can be observed in other files as well.